app/assets/stylesheets/components/_player.scss

```diff
@@ -127,7 +128,7 @@
 
 #player-buttons-2 {
     float:right;
-    margin:12px 10px 0 0;
+    margin:2px 10px 0 0;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space